CreateClothingPRP.py

```diff
@@ -245,7 +245,7 @@ def CreatePage(input_path: Path, output_path: Path, gcAgeInfo: plAgeInfo, pageIn
             #print(f" Adding element #{idx} named '{element['name']}'")
             ci.addElement(element["name"])
             for layeridx in element["layers"]:
-                texSpec = element["layers"][layeridx]
+                texName = texSpec = element["layers"][layeridx]
```

Comment:
This isn't very Pythonic. Instead, we should probably change the `else` on line 262 to `elif isinstance(texSpec, str)` and set `textName` there. An `else:` should be added after that to `raise ValueError`